wasm/src/account/view_key.rs

```diff
@@ -15,7 +15,7 @@
 // along with the Aleo library. If not, see <https://www.gnu.org/licenses/>.
 
 use crate::account::{Address, PrivateKey};
-use aleo_account::{Record, ViewKey as ViewKeyNative};
+use aleo_account::{CipherTextRecord as CipherTextRecordNative, ViewKey as ViewKeyNative};
```

Comment:
```suggestion
use aleo_account::{RecordCiphertext, ViewKey as ViewKeyNative};
```
We do not need the alias "RecordCiphertextNative" here because RecordCiphertext is not redefined in this wasm file.

Reply:
I was thinking that keeping the alias might look the most consistent. I agree though, better format to only type alias when strictly necessary